logistics/preparation.md

```diff
@@ -0,0 +1,109 @@
+# Preparation
+
+1. Identify the Preparation Owner from the Partner/Customer:
```

Comment:
Could also add the following points:
1. Although this is a bit of information, it is worth looking at it because it will ensure a smooth running hackathon
2. The hackathon organizer reading this page will have a much better understanding about Azure and Azure OpenAI Service + limits etc.